tools/modbus-slave/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.16)
+LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/submodules/cmake)
+
+include(projectheader)
+project_header(modbus-cpp)
+
+include(compiler)
+
+include_directories(
+    ${CMAKE_SOURCE_DIR}/include
+)
+
+set(SRC_LIST
+    ${CMAKE_CURRENT_SOURCE_DIR}/modbusbase.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/modbusbase.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/modbustcp.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/modbustcp.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/modbusrtu.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/modbusrtu.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/connectionconfig.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/connectionconfig.cpp
```

Comment:
Bit weird to have this in the tools folder, while it includes the `src` files? No clue what the actual intention is of these files .. `tools`.. *shrug* :D

Reply:
Still a WIP... Didn't figure this one out / thought this through yet...